src/prefect/agent/agent.py

```diff
@@ -47,12 +55,46 @@ def start(self) -> None:
         The main entrypoint to the agent. This function loops and constantly polls for
         new flow runs to deploy
         """
+        tenant_id = self.agent_connect()
+        while True:
+            self.agent_process(tenant_id)
+            time.sleep(self.loop_interval)
+
+    def agent_connect(self) -> str:
+        """
+        Verify agent connection to Prefect Cloud by finding and returning a tenant id
+
+        Returns:
+            - str: The current tenant id
+        """
+        print(ascii_name)
         self.logger.info("Starting {}".format(type(self).__name__))
+        self.logger.info(
+            "Agent documentation can be found at https://docs.prefect.io/cloud/agent"
+        )
         tenant_id = self.query_tenant_id()
```

Comment:
I believe this line will ValueError if no tenant is found because it'll try to access element `[0]` of an empty list; you may want to trap the error in `query_tenant_id()` and return None